src/components/SquareImageBlock.vue

```diff
@@ -1,14 +1,25 @@
 <template>
-  <div @mouseover="hover = true" @mouseleave="hover = false" class="relative box text-white">
-    <a :href="target_url" target="_blank">
-      <div class="z-0 absolute darken top-0 left-0">
-        <img class="object-cover h-64" :class="{ lighter: hover }" :src="imagePath()" alt="image" />
+  <div
+    @mouseover="hover = true"
+    @mouseleave="hover = false"
+    class="relative apply m-4 p-3 text-center text-white box-size"
+  >
+    <div class="z-0 absolute darken top-0 left-0">
+      <img
+        v-if="this.hover_enabled"
```

Comment:
I'm surprised the `v-if` attribute requires `this.` scoping. I'd have thought you could just use:
`v-if="hover_enabled"`

Reply:
Was just an old habit creeping back in from my C++ days.  T_T one day I will break free from it. Definitely works as you thought it would without it.